webapp/jest.config.js

```diff
@@ -26,8 +26,10 @@ module.exports = {
   moduleNameMapper: {
     '\\.(svg)$': '<rootDir>/test/fileMock.js',
     '\\.(css|less)$': 'identity-obj-proxy',
+    '@mapbox/mapbox-gl-geocoder': 'identity-obj-proxy',
```

Comment:
I would never figured this out unless there is stackoverflow for this. 💪🏻 

Sorry - useless comment - close pls

Reply:
This was my hack in total dispare